src/plugins/comicsPlayer/plugin.js

```diff
@@ -241,6 +241,9 @@ export class ComicsPlayer {
     }
 }
 
+// the comic book archive supports any kind of image format as it's just a zip archive
+const supportedFormats = ['jpg', 'png', 'avif', 'gif', 'bmp', 'dib', 'tiff', 'tif'];
```

Comment:
What about "jpeg"?

Reply:
> What about "jpeg"?

I just checked the file extensions on Wikipedia and it seems like `jpg` and `jpeg` are not the only possible file extensions for a jpeg image. :O
Anyway,  I just added the entire list of possible file extensions for jpeg to the list. ;)